### PR TITLE
Added nested where formula eg

### PR DESCRIPTION
**Pull request by huilinn**
https://github.com/workato/issues/issues/5580

### CODE REVIEW

File: formulas/array-list-formulas.md [hunk 1]
```diff
@@ -154,8 +153,23 @@ These rows will be expressed as a list of hashes:
 ]
 ```
 
+The following is a simple WHERE formula retrieving lead records with the company Delphi Chemicals.
+![Simple where formula retrieving lead records from company Delphi Chemicals](/assets/images/formula-docs/non-nested-where-formula.png)
+*Simple where formula retrieving lead records from company Delphi Chemicals*
+
+### Example of nested where formula
```
Comment (eeshan92) [inline formulas/array-list-formulas.md]: @huilinn more precise phrasing is where formula for nested fields. Saying nested where may cause users to think of using a where formula within another one. What do you think?

File: formulas/array-list-formulas.md [hunk 2]
```diff
@@ -154,8 +153,23 @@ These rows will be expressed as a list of hashes:
...
+The WHERE formula can be nested to filter for records by values not at the top level of the datatree.
```
Comment (eeshan92) [inline formulas/array-list-formulas.md]: Same here, we should explain that this section is to describe querying arrays of hashes based on a nested field

File: formulas/array-list-formulas.md (hunk 1 above)
Comment (huilinn) [reply formulas/array-list-formulas.md]: Sure, do you have any thoughts on what's a more precise term?

Comment (eeshan92) [reply formulas/array-list-formulas.md]: how about:
where formula for nested fields

**Comment by huilinn** [comment]
@eeshan92 have fixed comments

**Pull request merged**